index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <a href="#">My Logo</a>
+        
+        <nav>
+            <img src="images/Union.png" alt="menu">
+        </nav>
+    </header>
+
+    <section class="body">
+        <h1>I'm Mariane <br> Glad to see  you!</h1>
+
+        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+        <h2>LET'S CONNECT</h2>
+
+        <ul class="social">
+            <li>
+                <a href="#"><img src="images/Linkedinicon.png" alt="icons"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Vector1.png" alt="icons"></a>
+            </li>            <li>
+                <a href="#"><img src="images/Vector2.png" alt="icons"></a>
+            </li>            <li>
+                <a href="#"><img src="images/Vector3.png" alt="icons"></a>
+            </li>            <li>
+                <a href="#"><img src="images/Vector.png" alt="icons"></a>
```

Comment:
- [ ] The Social media icons are not showing on your page. Your image source name in the HTML doesn't match with your images in the image folder. To fix it, you should rename your images in the image folder also.

![s](https://user-images.githubusercontent.com/70483835/164943994-ec23ce33-8051-494e-9d6a-81cc4df09b71.png)

Reply:
> * [ ]  The Social media icons are not showing on your page. Your image source name in the HTML doesn't match with your images in the image folder. To fix it, you should rename your images in the image folder also.
> 
> ![s](https://user-images.githubusercontent.com/70483835/164943994-ec23ce33-8051-494e-9d6a-81cc4df09b71.png)

- [ ] Kindly work on it. :+1: